velox/type/Filter.h

```diff
@@ -55,18 +56,23 @@ enum class FilterKind {
   kHugeintRange,
 };
 
+class Filter;
+using FilterUniquePtr = std::unique_ptr<const Filter>;
```

Comment:
No need to `const` qualify the `Filter` here

Reply:
Fixed